azure-toolkit-libs/azure-toolkit-appservice-lib/src/main/java/com/microsoft/azure/toolkit/lib/appservice/function/FunctionAppBase.java

```diff
@@ -180,4 +178,12 @@ private boolean getIsRemoteDebuggingEnabled() {
                 StringUtils.equalsIgnoreCase(appSettings.get(JAVA_OPTS), getJavaOptsWithRemoteDebugEnabled(appSettings, appSettings.get(HTTP_PLATFORM_DEBUG_PORT)));
         return configEnabled && appSettingsEnabled;
     }
+
+    @Override
+    public boolean isEnableWebServerLogging() {
+        final OperatingSystem operatingSystem = Optional.ofNullable(getRuntime()).map(Runtime::getOperatingSystem).orElse(null);
+        final boolean isEnableApplicationLog = Optional.ofNullable(getDiagnosticConfig())
+                .map(DiagnosticConfig::isEnableApplicationLog).orElse(false);
+        return operatingSystem == OperatingSystem.LINUX || isEnableApplicationLog;
```

Comment:
seems wrong, need to confirm. 
only windows function with application log enabled supports streaming log.